bindings/python/synthizer.pyi

```diff
@@ -0,0 +1,260 @@
+from enum import Enum
+from typing import Iterator, List, Optional, Tuple, Union
+
+StringOrBytes = Union[str, bytes]
+Double3 = Tuple[float, float, float]
+Double6 = Tuple[float, float, float, float, float, float]
+EventSourceType = Union['Source', 'Generator']
+
+
+class LoggingBackend(Enum):
+    STDERR: int
+
+
+STDERR: LoggingBackend = LoggingBackend.STDERR
```

Comment:
You do this on all the enums, where you lift the constant out of the enum as well as putting it under the enum.  Given that Synthizer is `synthizer.LogLevel.STDERR` etc. aren't these top-level definitions incorrect/redundant?

Reply:
Given that:

```python
synthizer.ERROR is synthizer.LogLevel.ERROR
```

is `True`, I thought they were OK.